beginner_source/maskedtensor_overview.rst

```diff
@@ -0,0 +1,244 @@
+MaskedTensor Overview
+=====================
+
+This tutorial is designed to serve as a starting point for using MaskedTensors
+and discuss its masking semantics.
+
+Using MaskedTensor
+++++++++++++++++++
+
+Construction
+------------
+
+There are a few different ways to construct a MaskedTensor:
+
+* The first way is to directly invoke the MaskedTensor class
+* The second (and our recommended way) is to use :func:`masked.masked_tensor` and :func:`masked.as_masked_tensor` factory functions,
```

Comment:
is this still true that this is preferred to MaskedTensor?

Reply:
Discussed offline, yes it's still true